tests/unit/resources/catalog/test_mixins.py

```
@@ -467,3 +501,111 @@ async def test_async_document_create_with_file(async_document_service):  # noqa:
     assert "multipart/form-data" in request.headers["Content-Type"]
     assert result.to_dict() == response_data
     assert isinstance(result, DummyModel)
+
+
+def test_sync_update_file(update_file_service):  # noqa: WPS210
```

Ruff (0.14.6)

Unused `noqa` directive (unknown: `WPS210`)

Remove unused `noqa` directive

(RUF100)